client/containers/DashboardSubmissionWorkflow/types.ts

```diff
@@ -1,5 +1,19 @@
 import { SubmissionWorkflow } from 'types';
 
+export const submissionWorkflowConfigSteps = [
+	'instructions-requirements',
+	'response-emails',
+	'layout-banner',
+] as const;
```

Comment:
This is weird to see, but only because it is exported and used elsewhere (namely, in `submissionWorkflowStepPicker`.)
Just strange to see something like `import { variableNotAType } from './types';`

Reply:
I agree it's a bit weird but there's some precedent for it:

![image](https://user-images.githubusercontent.com/2208769/160711057-8fc77a92-df1c-4cf0-84c5-12650057ab8f.png)

Reply:
Probably going to keep it here, but I agree it's a big hmmm